plugins/plugin-chart-echarts/src/Timeseries/controlPanel.tsx

```diff
@@ -338,6 +338,19 @@ const config: ControlPanelConfig = {
             },
           },
         ],
+        [
+          {
+            name: 'rich_tooltip',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Rich Tooltip'),
```

Comment:
Should we be following the new conventions? As in, "Rich tooltip" instead of "Rich Tooltip".